docs/transaction.md

```diff
@@ -2,17 +2,11 @@
 
 ## UTXO overview
 
-**TODO maybe system/model instead of structure?**
-Mintlayer uses the Bitcoin UTXO structure instead of the account-based models of Ethereum, Ripple, Stellar, and others.
-**TODO need explanation of this sentence, it is not clear to me**
-Since each transaction's output is stored separately (even when sent to a single address), it is only possible to spend the entire transaction’s output.
+Mintlayer uses the Bitcoin UTXO system instead of the account-based models of Ethereum, Ripple, Stellar, and others. There are three essential reasons for this: 
```

Comment:
I'd say Bitcoin-esque as it's not identical to Bitcoin's.

Reply:
Changed it.